src/frontends/onnx/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenVINO ONNX Frontend
+
+The main responsibility of the ONNX Frontend is to import ONNX models and convert them into the `ov::Model` representation. 
+Other capabilities of the ONNX Frontend:
+* modification of tensors properties (like data type and shapes)
+* changing topology of the models (like cutting subgraphs, inserting additional inputs and outputs)
+* searching the models in a user-friendly way via tensors and operators names
+
+The component is written in `C++`. `Python` bindings are also available.
+If you want to contribute to the ONNX Frontend, follow [the common coding style rules](../../../docs/dev/coding_style.md).
+
+
+## Key contacts
+
+In case of any questions, review and merge requests, contact the [openvino-onnx-frontend-maintainers](https://github.com/orgs/openvinotoolkit/teams/openvino-onnx-frontend-maintainers) group.
+
+
+## Components
+
+ONNX Frontend implements an interface common to all frontends defined in the [Frontends API](../common/include/openvino/frontend).
+For backward compatibility reasons, the ONNX importer API (more lower-level abstraction approach) is still maintained. You can find it in the [ONNX Importer](./frontend/include/onnx_import/onnx.hpp).
+
+Read [How to add a new operation](./docs/how_to_add_op.md) page for more information on adding new operations to the ONNX Frontend and extending the existing ones. 
```

Comment:
This sentence is not about components, maybe we shouldn't duplicate it here and keep only in the tutorials section.

Reply:
removed